lib/editor/util/__tests__/__snapshots__/map.js.snap

```diff
@@ -160,70 +125,30 @@ Object {
   "coordinates": Array [
     Array [
       Array [
-        -77.922135,
-        42.089099,
```

Comment:
The result of the `should delete the first stop` test implies that the last stop is deleted and the line is backwards!

Before:

<img width="241" alt="Screen Shot 2019-04-03 at 5 06 01 PM" src="https://user-images.githubusercontent.com/3112493/55520818-d1386b80-5632-11e9-8650-36ebc2052533.png">

After:

<img width="236" alt="Screen Shot 2019-04-03 at 5 05 52 PM" src="https://user-images.githubusercontent.com/3112493/55520816-ced61180-5632-11e9-8a6a-d134766a0471.png">